robot_smach_states/src/robot_smach_states/navigation/navigate_to_observe.py

```diff
@@ -14,25 +14,27 @@ class NavigateToObserve(NavigateTo):
     to the convex hull is used. Otherwise, the distance to the pose of the entity is used
 
     """
-    def __init__(self, robot, entity_designator, radius=0.7):
+    def __init__(self, robot, entity_designator, radius=0.7, margin=0.075):
```

Comment:
I think I would prefer inner and outer radius.

Reply:
I disagree. Often we just want a certain radius. So it is convenient if the margin has a default